crates/node-bindings/src/transformer.rs

```diff
@@ -1,11 +1,49 @@
+use napi::bindgen_prelude::Buffer;
 use napi::Env;
 use napi::JsObject;
 use napi::JsUnknown;
 use napi_derive::napi;
 
 #[napi]
 pub fn transform(opts: JsObject, env: Env) -> napi::Result<JsUnknown> {
-  let config: parcel_js_swc_core::Config = env.from_js_value(opts)?;
+  let config = parcel_js_swc_core::Config {
```

Comment:
@devongovett Any idea on why `from_js_value` fails to deserialize the value for wasm but not napi? The error was `expected $name, got -1`. 

Is it okay to commit manually constructing the struct like this rather than using serde?

Reply:
Did we change things in `Config` since the repl broke?

Reply:
Is it possible to log the value getting passed?

Reply:
Looks like that error comes from here: https://github.com/napi-rs/napi-rs/blob/8b6cc10d779f94601775910e90d22067987fb0f7/crates/napi/src/bindgen_runtime/js_values/arraybuffer.rs#L702

Could be due to the upgrade of napi-rs somehow being incompatible with napi-wasm. I will investigate.